experiments/snippets/kanbanBoards/TaskManagementKanban.xml

```diff
@@ -0,0 +1,1010 @@
+<skuidpage showsidebar="false" showheader="false" tabtooverride="Lead" personalizationmode="server">
+	<models>
+		<model id="Tasks_NotStarted" limit="20" query="true" createrowifnonefound="false" datasource="salesforce" sobject="Task" orderby="ActivityDate DESC NULLS LAST">
+			<fields>
+				<field id="RecordTypeId"/>
```

Comment:
This causes errors on orgs without Record Types enabled, and doesn't harm the functionality of the page, so can we ax this?
```suggestion
```